lottery/src/simple_tests.rs

```diff
@@ -0,0 +1,165 @@
+#[cfg(test)]
+extern crate std;
+#[cfg(test)]
+use std::println;
+
+use codec::Encode;
+use gstd::BTreeMap;
+use gtest::{Program, System};
+use lt_io::*;
+const USERS: &'static [u64] = &[3, 4, 5];
+
+fn init(sys: &System) {
+    sys.init_logger();
+
+    let ft = Program::current(&sys);
+
+    let res = ft.send_bytes_with_value(USERS[0], b"Init", 10000);
+
+    assert!(res.log().is_empty());
+}
+
+#[test]
+fn start_lottery() {
+    let sys = System::new();
+    init(&sys);
+    let lt = sys.get_program(1);
+
+    let res = lt.send(USERS[0], Action::StartLottery(20000));
+    assert!(res.log().is_empty());
+
+    lt.send(USERS[0], Action::LotteryState);
```

Comment:
You can check the lottery state here

Reply:
> You can check the lottery state here

check the lottery state:
lt.send(USERS[0], Action::LotteryState);

Reply:
Lottery state can only be checked in the program:
debug!("LotteryState: {:?}", lottery.lottery_state);